sdk/identity/azure-identity/src/test/java/com/azure/identity/AzureCliCredentialTest.java

```diff
@@ -148,4 +148,48 @@ public void testValidMultiTenantAuth() {
             .expectErrorMatches(e -> e instanceof ClientAuthenticationException)
             .verify();
     }
+
+    @Test
+    public void testSubscriptionAccepted() {
+        // setup
+        TokenRequestContext request
+            = new TokenRequestContext().addScopes("https://vault.azure.net/.default").setTenantId("newTenant");
```

Comment:
Is it important to this test that there be different tenants? Just want to narrow the test to what it is really targeting if possible.